src/diffpy/structure/tests/__init__.py

```diff
@@ -38,10 +38,15 @@ def testsuite(pattern=""):
     from itertools import chain
     from os.path import dirname
 
-    from pkg_resources import resource_filename
+    # Use importlib.resources for Python 3.9 and newer or importlib_resources for older versions
+    try:
+        from importlib import resources as importlib_resources
```

Comment:
let's not import as `importlib_resources` which is deprecated.

let's do it the other way around, so use `resources` and `import importlib_resources as resources`

Reply:
@sbillinge fixed. Just import importlib_resources now